### DISCUSSION TITLE
Borg Backup stopped working (due to unexpected shutdown)

### DISCUSSION THREAD

**Discussion by ghost** [nextcloud/all-in-one#2221] [Questions] [answered]
Server was down while Borg was integrity testing a backup. Since then, no more backups can be made and none can be restored.

How can this problem be fixed again?

Borg repair button does not work.

**Comment by szaimen** [comment, marked as answer]
Hi, it is indeed unexpected that the system gets shut down while a borg operation is still running. In that case you could install borg on the host and run `sudo borg break-lock /mnt/backup/borg` (if the borg repository is located in `/mnt/backup/borg`).
